lib/diagnostic.rb

```diff
@@ -29,31 +29,31 @@
 
 character = 'Jar Jar Binks'
 
-Diagnostic.interpolation = 'ANSWER is really a Sith Lord.'
+Diagnostic.interpolation = print '#{character} is really a Sith Lord.'
 
 # Numbers in Ruby
 #
 # What is the type of object in Ruby for decimal numbers? Write your answer in
 # a Ruby string.
 
-Diagnostic.decimal = 'your response here'
+Diagnostic.decimal = to_f
 
 # What is the type of object in Ruby for integer numbers? Write your answer in
 # a Ruby string.
 
-Diagnostic.integer = 'your response here'
+Diagnostic.integer = 'to_i'
```

Comment:
`to_i` will attempt to convert the object that it's invoked upon into an integer.  e.g., `'8'.to_i` will attempt to convert the string `'8'` into the integer `8`.  If `class` is invoked on an integer, (e.g., `8.class`) then it will return the `Fixnum` class.  The `Integer` class is a superclass of both `Fixnum` and `Bignum`.  Since this is the first Ruby diagnostic, I'll be lenient and accept both `Fixnum` and `Integer`, even if they're not properly capitalized.